packages/core/src/tools/heavyCustomerDataWarning.ts

```diff
@@ -0,0 +1,24 @@
+import { display } from './display'
+import { ONE_KIBI_BYTE } from './utils'
+
+// RUM batch bytes limit is 16KB
+// ensure that we leave room for other event attributes and maintain a decent amount of event per batch
+// 3KB (customer data) + 1KB (other attributes) * 4 (events per batch) = 16KB
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏operator priorities
```suggestion
// (3KB (customer data) + 1KB (other attributes)) * 4 (events per batch) = 16KB
```